style.css

```diff
@@ -0,0 +1,36 @@
+#main{
+  position: relative;
+  width: 400px;
```

Comment:
consider using relative measurements like `em` or `rem` for measurements, unless there's a specific reason not to.